foodloop-db-files/seeders/02_insertFood.sql

```diff
@@ -1,3 +1,4 @@
+BEGIN;
```

Comment:
what does this a `COMMIT;` at the EOF do?

Reply:
this is not needed, will remove it